### DISCUSSION TITLE
[Resolved] Deployments stuck in queue

### DISCUSSION THREAD

**Discussion by rjantoine** [vercel/community#3498] [Help] [answered]
### Summary

I have tried to deploy a change in my code and it's been stuck in queue for over 10 minutes, normally I wait no more than a few seconds. https://www.vercel-status.com doesn't seem to be reporting any issues. Suggestions?

### Steps to Reproduce

Push change to code
See it added to the queue
Wait......

**Comment by corymyslywec** [comment]
same issue for myself

**Comment by nicholas-mercer** [comment]
Same issue for me - tried cancelling and recreating, just gets stuck repeatedly.

**Comment by DanielEgelund** [comment]
Seems like an issue at Vercel and not your code :)

Mine is also stuck in queue.

**Comment by swarnava** [comment]
Hi,

We are working on this. This is due to our Upstream provider: https://www.vercel-status.com/incidents/md7nskdfmncw

**Reply by sloth2012** [reply]
same issue. still stuck!

**Comment by swarnava** [comment, marked as answer]
The incident is resolved now and all builds should be processed now. If you still facing issue, please contact us using https://vercel.com/help with project details!
